content/en/docs/setup/production-environment/tools/kops.md

```diff
@@ -86,6 +91,7 @@ curl -LO https://github.com/kubernetes/kops/releases/download/$(curl -s https://
 ```
 
 To download a specific version, replace the
+
```

Comment:
nit: Could line 93 continue as (and remove line 98):
To download a specific version of kops, replace the following portion of the command with the specific kops version.

```shell
example code
```
For example, to download kops v1.15.0: